src/components/plugins/MapQuestionnaire.js

```diff
@@ -0,0 +1,199 @@
+import Button from 'react-bootstrap/lib/Button';
+import BaseCommentForm from '../BaseCommentForm';
+import CommentDisclaimer from "../CommentDisclaimer";
+import FormControl from 'react-bootstrap/lib/FormControl';
+import FormGroup from 'react-bootstrap/lib/FormGroup';
+import React from 'react';
+import {injectIntl, FormattedMessage} from 'react-intl';
+import {alert} from '../../utils/notify';
+
+
+class MapQuestionnaire extends BaseCommentForm {
```

Comment:
This is the third copy-paste of the plugin iframe send message listen listener blah blah blah stuff. It should really be refactored into a common base or other helpers.